spec/models/court_spec.rb

```diff
@@ -326,46 +326,4 @@
       end
     end
   end
-
```

Comment:
Yes. They have been failing becauase CGI remove the method and forget to remove the test too.